src/App.vue

```diff
@@ -459,6 +462,18 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
         const companyResources = CompanyResources.find(x => x.entityType === this.entityType)
         if (companyResources) this.setCompanyResources(companyResources)
         else throw new Error('Invalid Entity Type')
```

Comment:
I'm wondering if this is working correctly, since I over-wrote the legal type in App.vue, which should result in an invalid type here (since I don't have your CP code yet), but there was not error and I saw this:

![image](https://user-images.githubusercontent.com/10002889/128077732-243c3984-0329-4b7e-a4ad-3eb1e20de7c1.png)

Reply:
I think this is a type error or error on a component. 

CP is a valid type but it's resource file was just bare bones. So missing data is most likely causing an error rendering some components.
This IS fixed with the update resource file for Coops.

Reply:
Ah, the placeholder data existed for CP. OK, np then.